apps/client/src/plugins/Coordinates/CoordinatesTransformRow.js

```diff
@@ -67,6 +80,103 @@ class CoordinatesTransformRow extends React.PureComponent {
     });
   }
 
+  getCoordinates(title) {
+    let inputX, inputY;
+    // We find the specified numericFormat X and Y value
+    inputX = document.getElementsByName(`${title}numberformatX`)[0].value;
+    inputY = document.getElementsByName(`${title}numberformatY`)[0].value;
+    // And remove all blank spaces
+    inputX = inputX.replace(/\s/g, "");
+    inputY = inputY.replace(/\s/g, "");
+
+    return { inputX, inputY };
+  }
+
+  handleCopyToClipBoard(coordinateFormatTitle) {
+    //We get the correct input fields values using the title of the coordinate system
+    const { inputX, inputY } = this.getCoordinates(coordinateFormatTitle);
+
+    // We check if the values have any numbers, and if not exit the function early...
+    // and give an alert to the user
+    if (inputX === "" || inputY === "") {
+      // Display a message if any of the fields are empty
+      this.props.enqueueSnackbar("Kopiering misslyckades, fälten är tomma", {
+        variant: "error",
+      });
+      return;
+    }
+
+    // Set the string to be copied from the two X and Y values
+    const coordinatesString = `${inputX},${inputY}`;
+
+    // We create a temporary element to store the and copy the coordinateString
+    const input = document.createElement("input");
+    input.value = coordinatesString;
+
+    // Make the element non-interactive and hide it
+    input.setAttribute("readonly", "");
+    input.style.position = "absolute";
+    input.style.left = "-9999px";
```

Comment:
Instead of creating an element to copy from, wouldn't `navigator.clipboard.writeText(coordinatesString)` get the job done? There might be security considerations that i am missing though.